server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -44,6 +44,10 @@ object CromwellEntryPoint extends GracefulStopSupport {
 
   val gracefulShutdown = config.as[Boolean]("system.graceful-server-shutdown")
 
+  // 3 minute DNS TTL down from JVM default of infinite [BA-6454]
+  val dnsCacheTtl = config.getOrElse("system.dns-cache-ttl", 3 minutes)
+  java.security.Security.setProperty("networkaddress.cache.ttl", dnsCacheTtl.toSeconds.toString)
```

Comment:
TOL: might make you regret following Miguel's lead... is it worth having a test that this is being passed through correctly?

Reply:
I found tests like `cromwell.core.LoadConfigSpec` but I wasn't crazy about them, seems like we're mostly just testing Typesafe's `ConfigFactory`

The single workflow runner approach didn't end up going anywhere either because of course, we only run a single workflow and the JVM resets 🤦‍♂️ 

It's hard to argue I haven't exceeded my "is there an easy test" timebox at this point